hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/HandleWorkflowModule.java

```diff
@@ -56,11 +56,6 @@ static Expansion.CryptoSigsCreation provideCryptoSigsCreation() {
         return PlatformSigOps::createCryptoSigsFrom;
     }
 
-    @Provides
```

Comment:
Are we sure this is never needed?  It's a DI object provider, so might not be obvious.

Reply:
I didn't delete this. I moved it from the handle module to the preHandle module because it's also needed there, so it's instantiated earlier